### PR TITLE
feat:update user list css

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@MayueCif PTAL

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/Staff.razor.cs [hunk 1]
```diff
@@ -60,7 +60,24 @@ public int PageSize
...
-    public List<StaffDto> Staffs { get; set; } = new();
+    private List<StaffDto> _staffs = new();
+    public List<StaffDto> Staffs
+    {
+        get
+        {
+            return _staffs;
+        }
+        set
+        {
```
Comment (wuweilaiya) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/Staff.razor.cs:71]: 如果其他人用的是Staffs.AddRange是不是会有隐患，这个ui上的逻辑建议放在Razor里

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/Staff.razor.cs [hunk 2]
```diff
@@ -60,7 +60,24 @@ public int PageSize
...
+            _staffs = value?.Select((item) =>
+            {
+                item.Department = string.IsNullOrEmpty(item.Department) ? "-" : item.Department;
+                item.Position = string.IsNullOrEmpty(item.Position) ? "-" : item.Position;
+                return item;
+            })
+            .ToList() ?? new List<StaffDto>();
+        }
+    }
```
Comment (wuweilaiya) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/Staff.razor.cs:80]: 如果其他人用的是Staffs.AddRange是不是会有隐患，这个ui上的逻辑建议放在Razor里

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/ThirdPartyUser.razor [hunk 3]
```diff
@@ -56,11 +56,11 @@
                     case (nameof(ThirdPartyUserDto.IdpDetailDto)):
                         if (context.Item.IdpDetailDto.Name == LdapConsts.LDAP_NAME)
                         {
-                            <MIcon Size="40">@context.Item.IdpDetailDto.Icon</MIcon>
+                            <MIcon Size="24" Style="color: #323D6F !important;">@context.Item.IdpDetailDto.Icon</MIcon>
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/ThirdPartyUser.razor:59]: #323D6F  not recommended,replace class in stack component

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/User.razor [hunk 4]
```diff
@@ -1,82 +1,82 @@
...
-        <div class="d-flex">
-            <MSpacer />
-            <div>
-                <SUserAutoComplete Small FillBackground=false @bind-Value="UserId" />
-            </div>     
-            <div @onclick="() => Filter = !(Filter ?? false)" class="ml-4 d-flex button white">
-                <MIcon Class="ma-auto" Color="emphasis2" Size=20>
-                    @(Filter is true ? IconConstants.FilterOff : IconConstants.Filter)
-                </MIcon>
+<div class="d-user">
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/User.razor:4]: what d-user affect?

File: src/Web/Masa.Auth.Web.Admin.Rcl/wwwroot/css/masa-auth.css [hunk 5]
```diff
@@ -966,3 +967,21 @@ input::-webkit-input-placeholder {
 .m-btn-group .m-item-group .m-btn.m-btn {
     padding: 0px 28px !important;
 }
+
+.d-flex table {
+    /*height:100%;*/
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Admin.Rcl/wwwroot/css/masa-auth.css:972]: delete

**Review by MayueCif — COMMENTED** [review]
@kogel-net

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/User.razor (hunk 4 above)
Comment (kogel-net) [reply src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/User.razor:4]: Use as style penetration

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/ThirdPartyUser.razor (hunk 3 above)
Comment (kogel-net) [reply src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/ThirdPartyUser.razor:59]: ok

**Review by MayueCif — APPROVED** [review]

**Review by MayueCif — APPROVED** [review]

**Pull request merged**